deploy/helm-charts/webhook/templates/validating-webhook.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
```

Comment:
admissionregistration.k8s.io/v1 is available, should we not use that?

Reply:
noted.